apps/prairielearn/elements/pl-matching/pl-matching.css

```diff
@@ -47,3 +47,38 @@
   background-color: rgba(0, 0, 0, 0.03);
   margin-right: 0.5rem;
 }
+
+.pl-matching-container-right {
+  display: flex;
+}
+
+.pl-matching-container-right .pl-matching-statements {
+  flex: 0.2 0 25%;
+  display: flex;
+  flex-direction: column;
+}
+
+.pl-matching-container-right .pl-matching-options {
+  flex: 0 0 35%;
+  display: flex;
+  flex-direction: column;
+}
+
+.pl-matching-container-bottom {
+  display: block;
+}
+
+.pl-matching-container-bottom .pl-matching-statements {
+  display: flex;
+  flex-direction: column;
+}
+
+.pl-matching-container-bottom .pl-matching-options {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Oh okay, I've removed those unnecessary classes. 

I'm only keeping this part for the functionality:
.pl-matching-container-bottom {
  display: block;
}
And this part because the option value seems to be partially hidden within the dropdown in this usecase:
.pl-matching-container-bottom .pl-matching-statement-select,
.pl-matching-container-right .pl-matching-statement-select {
  min-width: 4.5em;
}